### PR TITLE
docs: revert a doc format change to reenable slurm tests

### PR DESCRIPTION
**Pull request by hamidzr**
## Ticket

https://hpe-aiatscale.atlassian.net/browse/DET-10292

## Description

https://hpe-aiatscale.slack.com/archives/C04C9JXB1C2/p1715354349333909

## Test Plan

take out branch protections to let the tests run slurm tets should not fail with image not found
https://app.circleci.com/pipelines/github/determined-ai/determined/55541/workflows/baba4a8f-bd5c-4fd4-9b9e-d99fc065b5c4/jobs/2563937 test ran and passed I'll put back in the branch protection.

## Checklist

- [x] Take out branch protection removal

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/real_config.yml [hunk 1]
```diff
@@ -4231,10 +4231,10 @@ workflows:
           name: test-e2e-slurm-misconfigured
           requires:
             - package-and-push-system-local-ee
-          filters:
-            branches:
-              only:
-                - main
+          # filters:
```
Comment (JComins000) [inline .circleci/real_config.yml:4234]: im not gonna approve until comments are gone

Comment (hamidzr) [reply .circleci/real_config.yml:4234]: those are there to just make it easier for the reviewer to see the tests pass.

Comment (hamidzr) [reply .circleci/real_config.yml:4234]: I can just link to the CI run once it finishes

Comment (JComins000) [reply .circleci/real_config.yml:4234]: maybe leave it in draft while you're testing. infra robot is pinging me for an automatic review even though the PR isn't quite ready

**Review by JComins000 — APPROVED** [review]

**Pull request merged**